documentation/src/main/asciidoc/topics/con_session_externalization.adoc

```diff
@@ -0,0 +1,21 @@
+[id='session-externalization_{context}']
+= Session externalization
+
+{brandname} can provide external caches for applications built on Red Hat JBoss Enterprise Application Platform (EAP), Red Hat Build of Quarkus, Red Hat JBoss Web Server (JWS), and Spring.
+These external caches store HTTP sessions and other data independently of the application layer.
+
+.Session externalization
+//Community content
+ifdef::community[]
+image::session-externalization-ispn.png[This illustration depicts a {brandname} cluster that offloads session data from applications.]
+endif::community[]
+//Downstream content
+ifdef::downstream[]
+image::session-externalization.png[This illustration depicts a {brandname} cluster that offloads session data from applications.]
+endif::downstream[]
+
+Externalizing sessions to {brandname} gives you the following benefits:
+
+Elasticity:: Make applications stateless to add and remove nodes without the need for rebalancing operations as well as to avoid downtime during cluster upgrades.
```

Comment:
Clarification: externalizing sessions does not make the _application_ stateless, it makes the _application server_ stateless (or, at least, less stateful).  The core advantage is eliminating the need for rebalancing as a consequence of scaling the application server layer.  Generally, downtime should never an issue (provided the application server cluster is managed correctly)

Reply:
Thanks again @pferraro I've reworded with this clarification.